types/params.go

```diff
@@ -67,6 +67,11 @@ type EvidenceParams struct {
 	// mechanism for handling [Nothing-At-Stake
 	// attacks](https://github.com/ethereum/wiki/wiki/Proof-of-Stake-FAQ#what-is-the-nothing-at-stake-problem-and-how-can-it-be-fixed).
 	MaxAgeDuration time.Duration `json:"max_age_duration"`
+
+	// Ratio for the max amount of evidence relative to the amount of validators to be included in a block
+	// i.e. MaxNumEvidence = ValSize * MaxEvidenceToValidatorsRatio
+	// where MaxNumEvidence is always rounded up. Default is 1/3 of validator size
+	MaxNumEvidence int64 `json:"max_num_evidence"`
```

Comment:
can we make this `uint16`?

Reply:
sure, I agree - does that mean I can remove the greater than 0 catch as well?

Reply:
the only problem I see with uint16 is that there will most likely need to be quite a few conversions

Reply:
Also I just discovered this: 
```
json: cannot unmarshal string into Go value of type uint16)
```

Reply:
uint32 should work. could we try this?

Reply:
sure I can give it a go

Reply:
this should be added to the abci consensusparams as well https://github.com/tendermint/tendermint/blob/master/abci/types/types.proto#L298